spec/features/guest/show_spec.rb

```diff
@@ -0,0 +1,48 @@
+require "rails_helper"
+
+RSpec.describe Guest do
+  it "shows the guest show page including guest name the rooms they stayed in including the room's suite, nightly rate and the name of the hotel" do
+    @guest1 = Guest.create!(name: "George Russell", nights: 2 )
+    @hotel1 = Hotel.create!(name: "Echo Mountain Inn", location: 'Echo Mountain')
+    @room1 = Room.create!(rate: 100, suite: 'Standard', hotel_id: "#{@hotel1.id}")
+    GuestRoom.create!(guest: @guest1, room: @room1)
+    
+    visit "/guests/#{@guest1.id}"
+    expect(page).to have_content("George Russell")
+    expect(page).to have_content("100")
```

Comment:
This assertion: `expect(page).to have_content(125)`, is not that great of an assertion. Is `100` supposed to be part of a phone number, address, something else? You should be more specific with this test by either using a `within` block or anchor text.

`expect(page).to have_content("Rate: 100")`

I am not too worried about text data causing too much of an ambiguity problem, but numbers are more like to return false positives.